calculator/operations.py

```diff
@@ -1,18 +1,24 @@
 
```

Comment:
A lot of the variable names can be more descriptive. For example `*args` might be better replaced with `*numbers`. Same with `m` and `x` in down `def multiply(*args)`. It helps with readability for others, as well as yourself when you're reviewing your code.